runtime/gc_base/ContinuationObjectList.cpp

```diff
@@ -46,13 +46,22 @@ MM_ContinuationObjectList::MM_ContinuationObjectList()
 }
 
 MM_ContinuationObjectList *
-MM_ContinuationObjectList::newInstanceArray(MM_EnvironmentBase *env, uintptr_t arrayElements)
+MM_ContinuationObjectList::newInstanceArray(MM_EnvironmentBase *env, uintptr_t arrayElements, MM_ContinuationObjectList *listsToCopy, uintptr_t listsToCopySize)
```

Comment:
how do you like naming the 2 integers similarly, like: arrayElementsTotal and arrayElementsToCopy?

Reply:
Agree!